sample-client-app/sample-app-with-shared-lib/src/main/java/com/vmware/vipclient/sample/Main.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2019-2020 VMware, Inc.
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+package com.vmware.vipclient.sample;
+
+import java.util.Locale;
+
+import com.vmware.vipclient.i18n.I18nFactory;
+import com.vmware.vipclient.i18n.VIPCfg;
+import com.vmware.vipclient.i18n.base.cache.FormattingCache;
+import com.vmware.vipclient.i18n.base.cache.MessageCache;
+import com.vmware.vipclient.i18n.base.instances.TranslationMessage;
+import com.vmware.vipclient.i18n.exceptions.VIPClientInitException;
+import com.vmware.vipclient.i18n.util.LocaleUtility;
+
+public class Main {
+	
+	public static void main(String[] args) {
+		Locale sharedLibLocale = Locale.forLanguageTag("es");
+		if (args.length > 0) {
+			// set locale
+			System.out.println("Set locale to: " + args[0]);
+			sharedLibLocale = Locale.forLanguageTag(args[0]);
+		}
+
+		// Initialize root application's VIP configuration
+		VIPCfg cfg = VIPCfg.getInstance();
+		try {
+			cfg.initialize("sampleconfig");
+		} catch (VIPClientInitException e) {
+			System.out.println(e.getMessage());
+		}
+		cfg.initializeVIPService();
+		cfg.createTranslationCache(MessageCache.class);
+		cfg.createFormattingCache(FormattingCache.class);
+
+		TranslationMessage tm = (TranslationMessage) I18nFactory.getInstance(cfg).getMessageInstance(TranslationMessage.class);
+		String rootMsg = tm.getMessage(Locale.forLanguageTag("fr"), "default", "global_text_username");
+		System.out.println("Root application's message; fetch is triggered from the root application:  " + rootMsg);
```

Comment:
In actual application, initialize code(line 28-37) usually is not put together(put in one method) with getMessage code(line 39-41). Usually the former is called only once when application start, and the latter is called many times at where it's needed. 
So I think put initialize code in a method, and get message code in another method is better.

Reply:
We can do this as future improvement.

Reply:
The point is to demonstrate how to pass the tmSharedLib in the simplest way possible, not how to design the root app or the shared lib.

Reply:
I think we should design sample app to make it easiest to use and as close to the real application as possible. So suggest future improvement.

Reply:
Yes, I agree. Actually, I think we should review the current design of shared library feature first.